### ISSUE TITLE
[A11y] The color pallet is not visible present for the "Font color "combo box after applying high contrast black and white mode.

### ISSUE THREAD

**Issue by lumills** [ckeditor/ckeditor4#4789]
## Type of report

Bug

1. Open any ckeditor environment leveraging the font plugin
2. Ensure the Font colors toolbar button is present
3. Apply high contrast black and white mode
4. TAB to the "Font color "combo box and expand it using ENTER key.
5. Observe that color pallet is not visible present for the "Font color "combo box after applying high contrast black and white mode.

### Expected result

The color pallet should be visible present for the "Font color "combo box after applying high contrast black and white mode.

### Actual result

The color pallet is not visible present for the "Font color "combo box after applying high contrast in both black and white mode.

User Impact:
The low vision users won't be able to see the color pallet present for the "Font color "combo box after applying high contrast black and white mode.

MAS Reference:
MAS 1.4.3 - Contrast (Minimum)

**Comment by Dumluregn** [comment]
Duplicate of https://github.com/ckeditor/ckeditor4/issues/4538.
